.github/workflows/fast_testing.yaml

```
@@ -0,0 +1,105 @@
+name: fast_testing
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  linux:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - 'debug-master'
+
+    env:
+      TNT_DEBUG_PATH: /home/runner/tnt-debug
+
+    runs-on: ubuntu-20.04
```

We could try to use a newer version.
```suggestion
    runs-on: ubuntu-24.04
```

---

Updated to 22.04. Our install script doesn't support 24.04.

---

> Updated to 22.04. Our install script doesn't support 24.04.

Are you about:

https://tarantool.io/XodVosJ/release/3/installer.sh ?

It's confusing.

---

Yep.

> We support the following versions of Ubuntu: 16.04, 18.04, 20.04, 22.04.